prompt_toolkit/key_binding/bindings/emacs.py

```diff
@@ -234,6 +244,133 @@ def _(event: E) -> None:
         """
         event.current_buffer.exit_selection()
 
+    def unshift_move(event: E):
```

Comment:
This needs a return annotation ``-> None:`` in order to tell mypy to typecheck this function.

Reply:
Can you add a docstring too for this function?